lib/rubocop/cop/style/request_referer.rb

```diff
@@ -0,0 +1,38 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks for uses of request.referrer.
+      class RequestReferer < Cop
+        include ConfigurableEnforcedStyle
+
+        def on_send(node)
+          if offense?(node)
+            add_offense(node.source_range, node.source_range, message)
+          end
+        end
+
+        def autocorrect(node)
+          ->(corrector) { corrector.replace(node, "request.#{style}") }
+        end
+
+        private
+
+        def offense?(node)
+          receiver = node.receiver
+          node.method_name == wrong_syntax && receiver.method_name == :request
+        end
+
+        def message
+          "Use `request.#{style}` instead of `request.#{wrong_syntax}`"
```

Comment:
Thanks @bbatsov but in line 25 there's a check `receiver.method_name == :request` which means the receiver it's gonna be `request` always.
If we change the receiver to be generic, it could affect user defined methods on other classes.